### PR TITLE
Fixed mvploop .

### PR DESCRIPTION
**Pull request by msvinaykumar**
Handled exception when metric results not collected due to some reasons.

Signed-off-by: msvinaykumar <vinakuma@redhat.com>

### CODE REVIEW

File: src/main/java/com/autotune/analyzer/loop/EMInterface.java [hunk 1]
```diff
@@ -1,75 +1,75 @@
...
-	 * @param autotuneExperiment
-	 * @param experimentTrial
-	 */
-	public static void SendTrialToEM(AutotuneExperiment autotuneExperiment,
-											  ExperimentTrial experimentTrial) {
-		try {
-			int trialNumber = experimentTrial.getTrialInfo().getTrialNum();
-			// Prepare to send the trial config to EM
-			String experimentTrialJSON = TrialHelpers.experimentTrialToJSON(experimentTrial);
+    private static final Logger LOGGER = LoggerFactory.getLogger(HPOInterface.class);
```
Comment (bharathappali) [inline src/main/java/com/autotune/analyzer/loop/EMInterface.java:24]: Can `HPOInterface.class` be changed to `EMInterface.class`?

Comment (msvinaykumar) [reply src/main/java/com/autotune/analyzer/loop/EMInterface.java]: done

File: src/main/java/com/autotune/analyzer/loop/EMInterface.java [hunk 2]
```diff
@@ -1,75 +1,75 @@
...
+        // TODO: Need to parse all the obj function results and calculate the result
+        try {
+            double reqSumMean = reqSum.getEmMetricResult().getEmMetricGenericResults().getMean();
+            double reqCountMean = reqCount.getEmMetricResult().getEmMetricGenericResults().getMean();
+            double rspTime = reqSumMean / reqCountMean;
+            LOGGER.info("Calculated rspTime (" + rspTime + ") = reqSumMean (" + reqSumMean + ") / reqCountMean (" + reqCountMean + ");");
+            experimentTrial.getTrialDetails().get(String.valueOf(trialNumber)).setResult(String.valueOf(rspTime));
+        } catch (Exception e) {
+            LOGGER.error("Error calculating response time due to {}", e.getMessage());
+            experimentTrial.getTrialDetails().get(String.valueOf(trialNumber)).setResult(null);
```
[the remark this answers is not in the document]
Comment (msvinaykumar) [reply src/main/java/com/autotune/analyzer/loop/EMInterface.java]: Agreed . Right now I checked links and few receiver end seems to be fine now .. However I left to do note.

**Review by dinogun — APPROVED** [review]
LGTM for the demo

**Pull request merged**